script.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pip install pipenv
+pipenv install && pipenv run python main.py
```

Comment:
@NaveenKundeti-IBS  Since this is a shell script please have all steps in separate lines

Reply:
Updated Sir @tomegathericon as per your suggestion.